app/models/manageiq/providers/physical_infra_manager.rb

```diff
@@ -1,5 +1,7 @@
 module ManageIQ::Providers
   class PhysicalInfraManager < BaseManager
+    has_many :physical_servers, :foreign_key => "ems_id", :class_name => "PhysicalServer"
```

Comment:
I think the `:class_name => "PhysicalServer"` is superfluous. @chrisarcand is that correct?